src/Maestro/tests/Maestro.ScenarioTests/MaestroScenarioTestBase.cs

```diff
@@ -736,6 +736,22 @@ public async Task<TemporaryDirectory> CloneAzDoRepositoryAsync(string repoName,
             return shareable.TryTake()!;
         }
 
+        public async Task<TemporaryDirectory> CloneRepositoryWithDarc(string repoName, string version, string reposToIgnore, bool includeToolset, int depth)
+        {
+            string sourceRepoUri = GetRepoUrl("dotnet", repoName);
+
+            using var shareable = Shareable.Create(TemporaryDirectory.Get());
+            string directory = shareable.Peek()!.Directory;
```

Comment:
I thought we had nullable off in arcade-services...  We shouldn't have/need this ! here.

Reply:
I don't know, I've just been reusing the base code that Alex set up for the initial test. I'll give it a shot.